LICENSE.md

```diff
@@ -16,23 +16,23 @@ Version 2.0.2
 
 **Dependency**: Software required for the Mod to execute, compile, or otherwise "work" correctly.
 
-**API**: Software intermediary for interaction with the Mod.
+**API**: Software which is part of the Mod that allows other software to interact with the Mod.
```

Comment:
Not included. Used. Big difference, especially when dealing with the "@Optional" annotation. Given that the purpose of defining API is to exclude things using the API from restrictions, I don't see what value your edit adds.